### PR TITLE
feat: migrate `apps/data-backend` from express to fastify

### PR DESCRIPTION
**Pull request by EjembiEmmanuel**

### CODE REVIEW

File: apps/data-backend/src/index.ts [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
 
-app.use("/", router);
-const port = process.env.PORT || 5050;
-app.listen(port, () => {
-  console.log(`🚀 Backend server running at http://localhost:${port}`);
-  try {
-    launchBot(process.env.TELEGRAM_BOT_TOKEN || "");
-  } catch (error) {
-    console.error("Error launching bot:", error);
+fastify.listen({ port: 3000 }, function (err, address) {
```
Comment (MSghais) [inline apps/data-backend/src/index.ts:20]: Can we use a process.env.PORT or a default value PORT to 5050, please?

File: apps/data-backend/src/index.ts [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
+  if (err) {
+    fastify.log.error(err);
+    process.exit(1);
   }
+
+  // try {
```
Comment (MSghais) [inline apps/data-backend/src/index.ts:26]: Please try also if the bot launch work too after

File: apps/data-backend/src/index.ts (hunk 1 above)
Comment (EjembiEmmanuel) [reply apps/data-backend/src/index.ts:20]: Okay

File: apps/data-backend/src/routes/indexer/deploy-launch.ts [hunk 3]
```diff
@@ -1,90 +1,104 @@
...
+        });
+        return;
       }
-    });
 
-    let statsLaunch = tokensData[0];
-    for (let launch of tokensData) {
-      // statsLaunch
+      const launches = await prisma.token_launch.findMany({
+        select: {
```
[the remark this answers is not in the document]
Comment (MSghais) [reply apps/data-backend/src/routes/indexer/deploy-launch.ts:82]: I'm gonna merge and fix it thank you

**Review by MSghais — APPROVED** [review]

**Pull request merged**